### PR TITLE
Update packages/graphics current

### PR DESCRIPTION
**Pull request by heitbaum**
libepoxy: update to 1.5.5
- https://github.com/anholt/libepoxy/releases/tag/1.5.5
- No change URI only and SHA256

kmscube: update to HEAD
- drm: grab correct handle for plane != 0
- Multiplanar formats can have different BOs for each plane, so call the
- correct GBM function to get the handle for a specific plane

cairo: update to 1.17.4
- bumping to 1.17.4.
- Also note maintainer update - https://gitlab.freedesktop.org/cairo/cairo/-/blob/156cd3eaaebfd8635517c2baf61fcf3627ff7ec2/NEWS

### CODE REVIEW

File: packages/graphics/lcms2/package.mk [hunk 1]
```diff
@@ -9,3 +9,4 @@ PKG_SITE="http://www.littlecms.com"
 PKG_URL="$SOURCEFORGE_SRC/lcms/lcms/${PKG_VERSION}/${PKG_NAME}-${PKG_VERSION}.tar.gz"
 PKG_DEPENDS_TARGET="toolchain tiff"
 PKG_LONGDESC="An small-footprint color management engine, with special focus on accuracy and performance."
+PKG_TOOLCHAIN="configure"
```
Comment (CvH) [inline packages/graphics/lcms2/package.mk:12]: ??

File: packages/graphics/libepoxy/package.mk [hunk 2]
```diff
@@ -9,10 +9,11 @@
 
 PKG_NAME="libepoxy"
 PKG_VERSION="1.5.5"
-PKG_SHA256="261663db21bcc1cc232b07ea683252ee6992982276536924271535875f5b0556"
+PKG_SHA256="5d80a43a6524a1ebdd0c9c5d5105295546a0794681853c636a0c70f8f9c658ce"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/anholt/libepoxy"
-PKG_URL="https://github.com/anholt/libepoxy/releases/download/${PKG_VERSION}/libepoxy-${PKG_VERSION}.tar.xz"
+#PKG_URL="https://github.com/anholt/libepoxy/releases/download/${PKG_VERSION}/libepoxy-${PKG_VERSION}.tar.xz"
```
Comment (CvH) [inline packages/graphics/libepoxy/package.mk:15]:
```suggestion
```

**Review by CvH — CHANGES_REQUESTED** [review]

**Comment by mglae** [comment]
Current visualization.projectm 3.3.0 depend on libprojectM 3.1.7. We should not go ahead without having issues.

**Comment by CvH** [comment]
3.1.7 -> 3.1.8 includes just 106 commits (:
yea better keep 3.1.7 for now

File: packages/graphics/lcms2/package.mk (hunk 1 above)
Comment (heitbaum) [reply packages/graphics/lcms2/package.mk]: Fixed.

**Comment by heitbaum** [comment]
> Current visualization.projectm 3.3.0 depend on libprojectM 3.1.7. We should not go ahead without having issues.

Done. Removed from this pull.

**Review by CvH — APPROVED** [review]

**Pull request merged**